src/libraries/System.Net.ServerSentEvents/src/System/Net/ServerSentEvents/SseParser_1.cs

```diff
@@ -418,7 +425,7 @@ private bool ProcessLine(out SseItem<T> sseItem, out int advance)
 #endif
                     NumberStyles.None, CultureInfo.InvariantCulture, out long milliseconds))
                 {
-                    ReconnectionInterval = TimeSpan.FromMilliseconds(milliseconds);
+                    _nextReconnectionInterval = ReconnectionInterval = TimeSpan.FromMilliseconds(milliseconds);
```

Comment:
Do we need to be concerned about the exception that'll happen if this overflows?

Reply:
Good question. Not sure what the best approach is here. We could either:

1. Throw a better exception message,
2. Normalize to `TimeSpan.MaxValue` or
3. Ignore the field altogether

Seems like option 3 is more compatible with SSE's general approach to handling invalid data.

Reply:
Sounds good.

Reply:
Done.